PlayCover/Views/App Views/PlayAppView.swift

```diff
@@ -181,14 +180,26 @@ struct PlayAppConditionalView: View {
     @Binding var selectedTextColor: Color
     @Binding var selected: PlayApp?
 
+    @ObservedObject var app: PlayApp
+
     @State var iconURL: URL?
-    @State var app: PlayApp
     @State var isList: Bool
 
     var body: some View {
         Group {
             if isList {
                 HStack(alignment: .center, spacing: 0) {
+                    let noPlayToolsImage = Image(systemName: "exclamationmark.triangle.fill")
```

Comment:
This layout works well for IPA Library, but when the icon only shows up occasionally, the extra spacing looks a little off.

Reply:
Do you mean this extra spacing?
<img width="1014" alt="Screenshot 2022-10-20 at 17 32 53" src="https://user-images.githubusercontent.com/58153205/197078353-345e6179-3eed-429e-8c41-46b582ca6689.png">

Reply:
Yeah

Reply:
Would it just be better to have it offset? (Also should the padding be smaller between the icon and the app image?)
<img width="1014" alt="Screenshot 2022-10-20 at 19 36 11" src="https://user-images.githubusercontent.com/58153205/197090762-9b625b4b-2018-4663-b0bf-3251aba4d751.png">

Reply:
is it not better to change the location of the sign to the trailing side of the game/app icon?

Reply:
@TheMoonThatRises I think just offsetting is good enough for now

Reply:
Make sure the padding is consistent tho